src/compiler/diagnosticMessages.json

```diff
@@ -3519,6 +3519,14 @@
         "category": "Error",
         "code": 7036
     },
+    "Create namespace objects for ECMAScript imports which are neither callable nor constructable, implies `allowSyntheticDefaultImports`.": {

```

Comment:
this seems hard to follow.. we a more user friendly description.. something that has interop, ES modules and commonjs.

@DanielRosenwasser thoughts?

Reply:
I've changed it to `Enables emit interoperability bewteen commonjs and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'.` - better?